datasets/emotion/emotion.py

```diff
@@ -44,6 +45,12 @@ def _info(self):
             supervised_keys=("text", "label"),
             homepage=_URL,
             citation=_CITATION,
+            task_templates=[
```

Comment:
this is here for demo purposes only and appears to be the reason why the tests are failing. i'll remove it if we decide to merge this PR